test-cases/message-intermediate-invalid-timer-event.bpmn

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_0wep1f0" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.2.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.0.0">
+  <bpmn:process id="task-for-timer" name="task-for-timer" isExecutable="true">
```

Comment:
```suggestion
  <bpmn:process id="message-intermediate-invalid-timer-event" name="message-intermediate-invalid-timer-event" isExecutable="true">
```
Since there are so many tests, which I'm used to also load in Zeebe for local testing,
I prefer the name of the process matches the file name.